src/components/ReactModalOverride.scss

```diff
@@ -0,0 +1,21 @@
+.ReactModal__Overlay, .ReactModal__Overlay--after-open {
+    z-index: 10;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(20, 48, 103, 0.10) !important;
+}
+
+.ReactModal__Content, .ReactModal__Content--after-open {
+  border: none !important;
+  max-width: 475px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: auto;
+  position: static !important;
+  padding: 0px !important;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
```

Comment:
Seems like a lot of these styles are here to mimic an MDC card. Is it possible to apply our own class to the Modal component instead?